### PR TITLE
[INLONG-7789][Dashboard] Support create stream fields by statement

### LINKED ISSUE #7789: [Feature][Dashboard] Support create stream fields by statement
### Description

At present, when creating a Stream field, it needs to be input one by one, which is relatively inefficient, especially when there are too many fields, the input will be very difficult.

image

image

In order to improve the efficiency of creating new Stream fields, the following three batch import methods are provided:
1. CSV: "Comma-Separated Values", is a file format for storing and exchanging data in a tabular structure, where each row represents a record and each column represents a field. The values in each row are separated by commas, and each row is separated by a new line.
2. Standard SQL: Specify the field name, field type, and field remarks through DDL statements
3. Excel: import Excel template
4. JSON： Input a JSON statement and specify the information of the field through three keys: Name, Type, and Comment. 

image

### PR DESCRIPTION
**Pull request by featzhang**
[INLONG-7789][Dashboard] Support creating stream fields by a statement

- Fixes #7789 

image

image

image

image

https://github.com/apache/inlong/pull/7424

newgif_2

### CODE REVIEW

File: inlong-dashboard/src/ui/components/EditableTable/index.tsx [hunk 1]
```diff
@@ -274,23 +304,300 @@ const EditableTable = ({
...
+    setPreviewData(null);
+  };
+  const handleClearJSON = () => {
+    setJsonData('');
+  };
+  const handleClearSQL = () => {
+    setSqlData('');
+  };
+  const handleDownloadExcelTemplate = () => {
+    // 下载Excel模板
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/components/EditableTable/index.tsx:368]: Please use English for annotation.

File: inlong-dashboard/src/ui/components/EditableTable/index.tsx [hunk 2]
```diff
@@ -274,23 +304,300 @@ const EditableTable = ({
...
+    message.success('Excel模板下载成功');
+  };
+  const handleUploadExcel = () => {
+    // 上传Excel数据并解析
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/components/EditableTable/index.tsx:372]: ditto

**Review by healchow — APPROVED** [review]

**Comment by dockerzhang** [comment]
@featzhang please add an English image example other than Chinese, thanks.

File: inlong-dashboard/src/ui/components/FieldParseModule/index.tsx [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+
+  const [statement, setStatement] = useState('');
+  const [previewData, setPreviewData] = useState<RowType[]>([]);
+
+  const handleCancel = () => {
+    onHide(); // call onHide callback when closing module
+  };
+
+  const handleFormat = () => {
+    // 实现handleFormat函数，对statement按照selectFormat的格式进行代码格式化。
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/components/FieldParseModule/index.tsx:72]: Please use English annotation.

File: inlong-dashboard/src/ui/locales/cn.json [hunk 4]
```diff
@@ -505,6 +505,17 @@
   "meta.Nodes.Kudu.DefaultSocketReadTimeoutMs": "等待Socket默认超时(ms)",
   "meta.Nodes.Kudu.StatisticsDisabled": "禁用统计信息收集",
   "components.EditableTable.NewLine": "新增一行",
+  "components.EditableTable.BatchAddField": "批量解析字段",
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/locales/cn.json:508]: `components.EditableTable.BatchParseField`?

File: inlong-dashboard/src/ui/locales/en.json [hunk 5]
```diff
@@ -505,6 +505,17 @@
   "meta.Nodes.Kudu.DefaultSocketReadTimeoutMs": "SocketReadTimeout(ms)",
   "meta.Nodes.Kudu.StatisticsDisabled": "DisabledStatistics",
   "components.EditableTable.NewLine": "New line",
+  "components.EditableTable.BatchAddField": "Batch add",
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/locales/en.json:508]: Ditto.

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**